### PR TITLE
Adjust kubectl docs to reflect plugin approach

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

- Adjust kubectl docs to reflect plugin approach

## Testing

Preview URLs:
- https://kubectl-plugin.botkube-docs.pages.dev/next/usage/executor/kubectl
- https://kubectl-plugin.botkube-docs.pages.dev/next/configuration/executor/kubectl/

## Related issue(s)

- https://github.com/kubeshop/botkube/issues/841

### CODE REVIEW

**Review by pkosiec — COMMENTED** [review]
Looks good, a few comments. Also please see the review https://github.com/kubeshop/botkube/pull/982 🙂

File: docs/usage/executor/kubectl.md
Comment (mszostok) [reply docs/usage/executor/kubectl.md:36]: but actually the defaults are in the code not values.yaml 🤔 so anyway we can have a glitch 🤔

File: docs/configuration/executor/helm.md
Comment (mszostok) [reply docs/configuration/executor/helm.md:15]: I duplicated the `sed` statement for `configuration` as well, hope that it will work as expected 🤔

File: docs/configuration/executor/kubectl.md
Comment (mszostok) [reply docs/configuration/executor/kubectl.md:64]: I just left only a few items, to make it inlined and more readable 👍

**Review by pkosiec — APPROVED** [review]
LGTM 👍 If possible, before merge, please regenerate Helm chart doc after merging PR on `botkube`. Thanks!

**Pull request merged**
